### ISSUE TITLE
NVDA announces "Grouping" even when it is turrned off!

### ISSUE THREAD

**Issue by gauravahir28** [nvaccess/nvda#16004]
### Steps to reproduce:
1. open NVDA settings.
2. select document formatting.
3. in the "Elements", uncheck the checkbox for "grouping".
  4. Visit [this example](https://www.w3.org/WAI/ARIA/apg/patterns/checkbox/examples/checkbox/)
  5. navigate to the checkbox example!

### Actual behavior:

NVDA announces "Sandwich Condiments grouping" 
### Expected behavior:

When "grouping" option is turrned off, NVDA should not announce the "Grouping".
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
NVDA 2023.3
#### Windows version:
Windows 11 23H2 
#### Name and version of other software in use when reproducing the issue:
Google chrome Version 120.0.6099.130 (Official Build) (64-bit)
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
did not try
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by josephsl** [comment]
Hi, if you don’t mind, can you try with NVDA 2024.1 beta 2 and Firefox and see if any of these variables make a difference? Thanks.

**Comment by gauravahir28** [comment]
No difference in 2024.1 Beta 2!
tried with Firefox as well with both the NVDA versions(2023.3 and 2024.1 beta 2)

**Comment by Adriani90** [comment]
Closing as duplicate of #10635.
